docs/user/man_pages/bodhi-manage-releases.rst

```diff
@@ -11,114 +11,12 @@ Synopsis
 Description
 ===========
 
-DEPRECATED: This utility has been deprecated. Please use ``bodhi releases`` instead.
+REMOVED: This utility has been removed. Please use ``bodhi releases`` instead.
```

Comment:
For when it's time to merge this PR, we should also delete this file. The more noticeable place to make a note of the files removal is the release notes (see ```docs/user/release_notes.rst```).

Reply:
There is a link in releases_notes.rst, so after removal the following info is shown after bdocs:

Warning, treated as error:
/home/vagrant/bodhi/docs/user/release_notes.rst:563:unknown document: man_pages/bodhi-manage-releases

Is it ok to change lines in release_notes.rst in legacy releases?